src/helpers/cookieHelper.ts

```diff
@@ -0,0 +1,35 @@
+export const setCookie = (name: any, value: any, exdays?: any): void => {
```

Comment:
Use a library like [js-cookie](https://www.npmjs.com/package/js-cookie) for this purpose.